modules/role/variables.tf

```diff
@@ -1,52 +1,53 @@
-variable "name" {
-  description = "The name prefix for these IAM resources"
-  type        = "string"
-}
-
-variable "build_state" {
-  description = "A variable to control whether resources should be built"
-  type        = "string"
-  default     = true
+variable "aws_account" {
+  description = "A list of AWS accounts allowed to use this cross account role.  Required if the aws_services variable is not provided."
+  type        = list(string)
+  default     = []
 }
 
 variable "aws_service" {
   description = "A list of AWS services allowed to assume this role.  Required if the aws_accounts variable is not provided."
-  type        = "list"
+  type        = list(string)
   default     = []
 }
 
-variable "aws_account" {
-  description = "A list of AWS accounts allowed to use this cross account role.  Required if the aws_services variable is not provided."
-  type        = "list"
-  default     = []
+variable "build_state" {
+  description = "A variable to control whether resources should be built"
+  type        = string
```

Comment:
we should probably update this to type `bool`, just to ensure people are passing in the right types